hydra-access-controls/app/models/ability.rb

```diff
@@ -1,4 +1,5 @@
 # Allows you to use CanCan to control access to Models
 class Ability
+  include Blacklight::AccessControls::Ability
```

Comment:
I wonder if this line should be moved into `Hydra::Ability`?  That would make the `super` calls in Hydra::Ability make more sense about what it's referring to.